tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/test_corrupted_data.c

```diff
@@ -212,42 +213,69 @@ ZTEST(suite_fast_pair_storage_corrupted, test_inconsistent_key_id)
 
 ZTEST(suite_fast_pair_storage_corrupted, test_drop_first_key)
 {
-	store_key(next_account_key_id(ACCOUNT_KEY_MIN_ID));
+	if (ACCOUNT_KEY_CNT < 2) {
+		store_key(ACCOUNT_KEY_MIN_ID + 1, false);
+	} else {
+		store_key(next_account_key_id(ACCOUNT_KEY_MIN_ID), false);
+	}
 
 	initialization_error_validate();
 }
 
 ZTEST(suite_fast_pair_storage_corrupted, test_drop_key)
 {
+	if (ACCOUNT_KEY_CNT - (IS_ENABLED(CONFIG_TEST_BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY) ?
+				1 : 0) < 2) {
+		ztest_test_skip();
+	}
+
 	uint8_t key_id = ACCOUNT_KEY_MIN_ID;
 
-	store_key(key_id);
+	store_key(key_id, true);
 
 	key_id = next_account_key_id(key_id);
-	store_key(key_id);
+	store_key(key_id, false);
 
 	key_id = next_account_key_id(key_id);
 	key_id = next_account_key_id(key_id);
-	store_key(key_id);
+	store_key(key_id, false);
 
 	initialization_error_validate();
 }
 
 ZTEST(suite_fast_pair_storage_corrupted, test_drop_keys)
 {
+	if (ACCOUNT_KEY_CNT - (IS_ENABLED(CONFIG_TEST_BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY) ?
```

Comment:
Shouldn't we always skip this test if owner AK is enabled? (or try to perform rollover with the first non-owner AK then?